.evergreen/run-ocsp-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java
+#                               "jdk5", "jdk6", "jdk7", "jdk8", "jdk9", "jdk11"
+
+JDK=${JDK:-jdk11}
+
+############################################
+#            Functions                     #
+############################################
+
+provision_ssl () {
+  echo "SSL !"
+
+  # We generate the keystore and truststore on every run with the certs in the drivers-tools repo
+  if [ ! -f client.pkc ]; then
+    openssl pkcs12 -CAfile ${CA_FILE} -export -in ${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem -out client.pkc -password pass:bithere
+  fi
+
+  cp ${JAVA_HOME}/lib/security/cacerts mongo-truststore
+  ${JAVA_HOME}/bin/keytool -importcert -trustcacerts -file ${CA_FILE} -keystore mongo-truststore -storepass changeit -storetype JKS -noprompt
```

Comment:
Fwiw, this is the command I used when testing locally
`keytool -import -trustcacerts -keystore ./cacerts-linux -alias ca_ocsp -file /home/vincent/mongod/mock-ocsp/ca_ocsp.pem -storepass changeit`

The command in the script seems to pretty much be the same